<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tunewidget</class>
 <widget class="QMainWindow" name="tunewidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1264</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Расширенная настройка системы</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_9">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>540</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Суммарная фаза</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="5">
          <widget class="QTableWidget" name="sumPhase">
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>40</number>
           </attribute>
           <row>
            <property name="text">
             <string>1</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>1</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pbFullPhaseLoad">
           <property name="text">
            <string>Загрузить из файла</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pbFullPhaseSave">
           <property name="text">
            <string>Сохранить в файл</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="pbFullPhaseToSul">
           <property name="text">
            <string>Записать в СУЛ</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QRadioButton" name="rbViewCode">
           <property name="text">
            <string>Код</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QRadioButton" name="rbViewDegrees">
           <property name="text">
            <string>Градус</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Собственная фаза</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QTableWidget" name="ownPhase">
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>40</number>
           </attribute>
           <row>
            <property name="text">
             <string>1</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>1</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pbOwnPhaseLoad">
           <property name="text">
            <string>Загрузить из файла</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pbOwnPhaseSave">
           <property name="text">
            <string>Сохранить в файл</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Расчетная фаза</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pbCorrPhaseLoad">
           <property name="text">
            <string>Загрузить из файла</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pbCorrPhaseSave">
           <property name="text">
            <string>Сохранить в файл</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string> Распределение 1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string> Распределение 2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string> Распределение 3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string> Распределение 4</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QPushButton" name="pbCorrPhaseCalc">
           <property name="text">
            <string>Рассчитать</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="4">
          <widget class="QTableWidget" name="corrPhase">
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>40</number>
           </attribute>
           <row>
            <property name="text">
             <string>1</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Дополнительная фаза</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="4">
          <widget class="QTableWidget" name="addPhase">
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>40</number>
           </attribute>
           <row>
            <property name="text">
             <string>1</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>1</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pbAddPhaseLoad">
           <property name="text">
            <string>Загрузить из файла</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pbAddPhaseSave">
           <property name="text">
            <string>Сохранить в файл</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QComboBox" name="addPhaseDistribution">
           <item>
            <property name="text">
             <string> Распределение 1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string> Распределение 2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string> Распределение 3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string> Распределение 4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string> Распределение 5</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QPushButton" name="pbAddPhaseCalc">
           <property name="text">
            <string>Рассчитать</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Управление</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Модуль</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QComboBox" name="cbModuleNumber"/>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Фокус, м</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="spbFocus">
              <property name="decimals">
               <number>6</number>
              </property>
              <property name="minimum">
               <double>0.005000000000000</double>
              </property>
              <property name="maximum">
               <double>200.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>0.225000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Длина волны, м</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QDoubleSpinBox" name="spbLambda">
              <property name="decimals">
               <number>6</number>
              </property>
              <property name="minimum">
               <double>0.005000000000000</double>
              </property>
              <property name="maximum">
               <double>200.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.001000000000000</double>
              </property>
              <property name="value">
               <double>0.008817000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Шаг по Х, м</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QDoubleSpinBox" name="spbdX">
              <property name="decimals">
               <number>6</number>
              </property>
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>200.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.010000000000000</double>
              </property>
              <property name="value">
               <double>0.002540000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Шаг по Y, м</string>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QDoubleSpinBox" name="spbdY">
              <property name="decimals">
               <number>6</number>
              </property>
              <property name="minimum">
               <double>0.001000000000000</double>
              </property>
              <property name="maximum">
               <double>200.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.050000000000000</double>
              </property>
              <property name="value">
               <double>0.004400000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Масштаб решетки</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="spbElementSize">
              <property name="value">
               <number>14</number>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Отображать:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QComboBox" name="cbDrawSelector">
              <item>
               <property name="text">
                <string>Собственная фаза</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Рассчетная фаза</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Дополнительная фаза</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Суммарная фаза</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Радиус векторы</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="4">
             <widget class="QPushButton" name="pbCalculateFik">
              <property name="text">
               <string>Расчет корректур</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>700</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="graphLayout">
         <property name="spacing">
          <number>1</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1264</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>5</number>
  </property>
  <property name="gridDeltaY">
   <number>5</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
